Uplift/Views/Home/GymDetailView.swift

```diff
@@ -118,20 +118,33 @@ struct GymDetailView: View {
             VStack(spacing: 4) {
                 Spacer()
 
-                switch gym.status {
-                case .closed:
-                    Text("CLOSED")
-                        .font(Constants.Fonts.h3)
-                        .foregroundStyle(Constants.Colors.closed)
-                case .open:
+                if viewModel.fitnessCenterIsOpen(gym: gym) {
                     Text("OPEN")
                         .font(Constants.Fonts.h3)
                         .foregroundStyle(Constants.Colors.open)
-                case .none:
-                    EmptyView()
+                        .padding(12)
+                } else {
+                    Text("CLOSED")
+                        .font(Constants.Fonts.h3)
+                        .foregroundStyle(Constants.Colors.closed)
+                        .padding(12)
                 }
```

Comment:
Can we use a ternary operator for this? Also, is the `.padding(12)` for centering the text since we are temporarily removing the "View Hours" button? If so, make sure to write a comment about the padding.